app/controllers/about_controller.rb

```diff
@@ -1,4 +1,30 @@
 class AboutController < ApplicationController
   def us
+    @octkit_client = current_user.octkit_client
   end
+
+  def enterprise?
+    enterprise_account_id.present?
+  end
+
+  def operate_repo_class
+    enterprise? ? 'EnterpriseGithubRepository' : 'GithubRepository'
+  end
+
+  def operate_github_repopsitory
+    @operate_github_repopsitory ||= operate_repo_class.classify.constantize
+  end
+
+  def octokit_client(oauth_token = nil)
+   return @octokit_client if @octokit_client.present? && oauth_token.nil?
```

Comment:
__[rubocop]__
* Use 2 (not 0) spaces for indentation. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Use only ascii symbols in identifiers. [:ref](https://github.com/bbatsov/ruby-style-guide#english-identifiers)

<!-- Comment by [SideCI](https://sideci.com) -->